R/step-subset.R

```diff
@@ -135,6 +135,47 @@ select.dtplyr_step <- function(.data, ...) {
   step_group(out, groups)
 }
 
+#' @importFrom dplyr relocate
+#' @export
+relocate.dtplyr_step <- function(.data, ..., .before = NULL, .after = NULL) {
+  vars <- set_names(.data$vars, .data$vars)
+  to_move <- unname(tidyselect::eval_select(expr(c(...)), data = vars))
+  groups <- .data$groups
+  if (length(to_move) == 0) {
+    return(.data)
+  }
+  .before <- enquo(.before)
```

Comment:
What about just doing `.before <- unname(tidyselect::eval_select(.before, vars))`? I think that would simplify the code quite a bit.

Reply:
Could you briefly elaborate what you expect to be simplified away? The code is only very lightly modified from dplyr's relocate, so I had been assuming it was structured as such for good reasons.

Reply:
Oh well if it matches dplyr, keep it like that. I have no recollection of what the dplyr code looks like 😆